README.md

```diff
@@ -63,6 +63,26 @@ To add a props file:
 </Project>
 ```
 
+## Adjusting severity
+In order to adjust the severity of certain rules it is possible to use a `.globalconfig` file.
```

Comment:
Please mention where to put this file. Mention that .editorconfig will not work (with a link to the reason why).

Reply:
Updated